Beams/Logic/vtkSlicerBeamsModuleLogic.cxx

```diff
@@ -209,9 +208,7 @@ void vtkSlicerBeamsModuleLogic::UpdateTransformForBeam(vtkMRMLScene* beamSequenc
   }
 
   //TODO: Use one IEC logic in a private scene for all beam transform updates?

```

Comment:
Remove

Reply:
Actually please add a todo instead of this:
`//TODO: beamSequenceScene is not used! Check what it is used for. The error message suggests that it is the main MRML scene, but its name says otherwise.